src/lib/polymer-bootstrap.html

```diff
@@ -20,12 +20,23 @@
     var userPolymer = window.Polymer;
 
     window.Polymer = function(prototype) {
-      var ctor = desugar(prototype);
-      // Polymer.Base is now chained to ctor.prototype, and for IE10 compat
+      // if input is a `class` (aka a function with a prototype), use the prototype
+      // remember that the `constructor` will never be called
+      if (typeof prototype === 'function') {
```

Comment:
Wondering about ES6 syntax and people wanting to use `extends` with, something. Think that is worth supporting? If so, seems like it'd go somewhere right in here.